src/mantarray_file_manager/file_writer.py

```diff
@@ -156,3 +165,150 @@ def migrate_to_latest_version(
         current_file_path = migrate_to_next_version(
             current_file_path, working_directory=working_directory
         )
+
+
+def h5_file_trimmer(
+    file_path: str,
+    from_start: int,
```

Comment:
it looks like there aren't any test cases where you don't trim anything from the start or the end.  I think in the Story we wanted that to be an option. Perhaps `from_start:int=0` to make it a kwarg and a test to confirm if only the `from_end` argument is supplied (and the converse for `from_start`)